src/components/Shared/apiMappings.js

```diff
@@ -713,5 +710,143 @@ export const apiMappings = {
     ]
   },
 
-  //HR filters not migrated
+  //HR filters
+
+  [ResourceIds.EmployeeFilter]: {
+    type: LOOKUP,
+    endpoint: EmployeeRepository.EmployeeFilters.snapshot,
+    parameters: {
+      _branchId: 0
+    },
+    firstField: 'reference',
+    secondField: 'name',
+    valueOnSelection: 'recordId',
+    displayFieldWidth: 1,
+    firstFieldWidth: '40%',
+    columnsInDropDown: [
+      { key: 'reference', value: 'Reference' },
+      { key: 'fullName', value: 'Name' }
+    ]
+  },
+  [ResourceIds.GovernmentOrganizationFilter]: {
+    type: COMBOBOX,
+    endpoint: SystemRepository.GovernmentOrganization.qry,
+    valueField: 'recordId',
+    displayField: ['reference', 'name'],
+    columnsInDropDown: [{ key: 'name', value: 'Name' }]
+  },
+  [ResourceIds.DivisionFilter]: {
+    type: COMBOBOX,
+    endpoint: companyStructureRepository.DivisionFilters.qry,
+    valueField: 'recordId',
+    displayField: ['reference', 'name'],
+    columnsInDropDown: [{ key: 'name', value: 'Name' }]
+  },
+  [ResourceIds.BranchFilter]: {
+    type: COMBOBOX,
+    endpoint: companyStructureRepository.BranchFilters.qry,
+    valueField: 'recordId',
+    displayField: ['reference', 'name'],
+    columnsInDropDown: [
+      { key: 'reference', value: 'Reference' },
+      { key: 'name', value: 'Name' }
+    ]
+  },
+  [ResourceIds.CompanyPosition]: {
+    type: COMBOBOX,
+    endpoint: companyStructureRepository.CompanyPositions.qry,
+    parameters: `_filter=&_size=1000&_startAt=0&_sortBy=recordId`,
+    valueField: 'recordId',
+    displayField: ['reference', 'name'],
+    columnsInDropDown: [{ key: 'name', value: 'Name' }]
+  },
+  [ResourceIds.DepartmentFilter]: {
+    type: COMBOBOX,
+    endpoint: companyStructureRepository.DepartmentFilters.qry,
+    parameters: `_filter=&_size=1000&_startAt=0&_type=0&_activeStatus=0&_sortBy=recordId`,
+    valueField: 'recordId',
+    displayField: ['reference', 'name'],
+    columnsInDropDown: [{ key: 'name', value: 'Name' }]
+  },
+  [ResourceIds.CertificateFilter]: {
+    type: COMBOBOX,
+    endpoint: EmployeeRepository.CertificateFilters.qry,
+    valueField: 'recordId',
+    displayField: ['reference', 'name'],
```

Comment:
here you added display reference and name but in the columnsInDropdown only name is that how it should be?